sre-recipes/sandboxctl

```diff
@@ -73,10 +78,29 @@ def get_recipes():
     recipes = get_valid_recipes(attributes)
     return recipes
 
+def restart_loadgen(pattern=LoadPattern.BASIC):
+    """
+    Restarts the load generator with the intent of deploying a
+    particular load pattern.
+    """
+    DEVNULL = open(devnull, 'w')
+    command = (
+        'sed -i -e'
+        ' "s/(basic|step):latest/step:latest/g"'
+        ' ../kubernetes-manifest/loadgenerator/locust-main-controller.yaml'
+        ' ../kubernetes-manifest/loadgenerator/locust-worker-controller.yaml'
+    )
+    subprocess.call(split(command), stdout=DEVNULL)
+    command = (
+        'kubectl apply'
+        ' -f ../kubernetes-manifest/loadgenerator/'
```

Comment:
Instead of re-writing the manifest files, you should be able to use `kubectl set image` to change the deployment on the cluster dynamically (and then run `kubectl delete pods` to refresh the pods). Tiffany and Nick did something similar with the environment variables elsewhere in this file